parallel-install/pkg/jobmanager/loggingJobs.go

```diff
@@ -0,0 +1,92 @@
+package jobmanager
```

Comment:
Curious should we need to merge this example or is it better to put inside an md file ?

Reply:
I can also put it in he md, but since this is also a job which is needed, it should stay